client/src/components/Portfolio.jsx

```diff
@@ -51,19 +51,35 @@ class Portfolio extends Component {
                 <th> Date changed </th>
               </tr>
             </thead>
-            <tbody>
-            {items.map((it, i) => (
-              <tr key={i} onClick={() => this.onSelectPortfolioItem({ ticker: it.symbol, companyName: it.company_name })} style={{ cursor: 'pointer' }}>
-                <td> {it.symbol} </td>
-                <td>
-                  {it.symbol === 'USD' ? '-' : this.formatQuantity(it.quantity, it.symbol)}
-                </td>
-                <td> {it.symbol === "USD" ? "-": `\$${this.formatQuantity(it.price_per_share, 'USD')}`} </td>
-                <td> ${it.symbol === "USD" ? this.formatQuantity(it.quantity, 'USD') : this.getStockValueNumber(it.quantity, it.price_per_share)}</td>
-                <td> {timeago.format(it.date_changed)} </td>
-              </tr>
-            ))}
+            {enableClick ? (
+              <tbody>
+              {items.map((it, i) => (
+                <tr key={i} onClick={() => this.onSelectPortfolioItem({ ticker: it.symbol, companyName: it.company_name })} style={{ cursor: 'pointer' }}>
```

Comment:
`cursor:'pointer'` should only be set if `enableClick` is true. Otherwise, this should not be set. (`cursor:pointer` gives the 'cursor' the look when you're hovering over something clickable e.g.a  button)

Reply:
ah damn. Ok! I'll fix that. Thanks for pointing it out